src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class RightTriangle extends Rectangle {
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
```

Comment:
The constructor should not have an empty line before it. Also, consider using separate variables for 'firstLeg' and 'secondLeg' before passing them to the constructor for better readability.